lib/common/models/TransactionModel.ts

```diff
@@ -8,8 +8,13 @@ export default interface TransactionModel {
   anchorString: string;
   transactionFeePaid: number;
 
-  // Normalized fee sohuld always be populated in core layer when core makes call to transactions endpoint.
-  // It may not be populated in blockchain service. This allows flexibility for the value to be computed on the spot or stored.
+  /**
+   * Normalized fee sohuld always be populated in core layer when core makes call to transactions endpoint.
+   * It may not be populated in blockchain service. This allows flexibility for the value to be computed on the spot or stored.
+   * To remove potentially dangerous behavior. Make a seperate model
+   * https://github.com/decentralized-identity/sidetree/issues/937
```

Comment:
Nit: add TODO for tooling/searching.
```suggestion
   * TODO: https://github.com/decentralized-identity/sidetree/issues/937
```